.github/workflows/run-test.yml

```diff
@@ -1,9 +1,18 @@
 name: Run Mergin Plugin Tests
 
-on: [push]
+on: 
+  push:
+  workflow_dispatch:
+    inputs:
+      MERGIN_CLIENT_VER:
```

Comment:
Let's not introduce old namings :) 
```suggestion
      API_CLIENT_VER:
```

Reply:
The same below "Mergin client version either ..."

Reply:
Renamed all the occurences I settled for `PYTHON_API_CLIENT_VER` in the end